src/index.html

```diff
@@ -9,6 +9,63 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="container">
+
+    <div class="logo">
+      <a href="#">
+        <img src="/images/logo.png" alt="Moyo-logo">
```

Comment:
не используй слеш для гитхаба
```suggestion
        <img src="images/logo.png" alt="Moyo-logo">
```

Reply:
можешь погуглить про относительный путь и какие бывают

Reply:
Дякую! Не знав що для гітхабу не треба слеш.